pkg/apis/velero/v1/backup.go

```diff
@@ -164,47 +199,57 @@ const (
 // BackupStatus captures the current status of a Velero backup.
 type BackupStatus struct {
 	// Version is the backup format version.
-	Version int `json:"version"`
+	// +optional
+	Version int `json:"version,omitempty"`
 
 	// Expiration is when this Backup is eligible for garbage-collection.
-	Expiration metav1.Time `json:"expiration"`
+	// +optional
+	Expiration metav1.Time `json:"expiration,omitempty"`
 
 	// Phase is the current state of the Backup.
-	Phase BackupPhase `json:"phase"`
+	// +optional
```

Comment:
should we be enumerating the enum values here?

Reply:
I decided to not do any validation for status fields as we don't expect them to be set by users - can go and add them if we think validation is needed here though.

Reply:
Even if it's not set by users, maybe it would be useful for catching bugs.

Reply:
Yeah I was thinking that, although it gives us less flexibility. But I think it is worth adding, so will update this.